test/conformance/helpers/broker_tracing_test_helper.go

```diff
@@ -77,7 +78,7 @@ func setupBrokerTracing(
 	client *lib.Client,
 	loggerPodName string,
 	tc TracingTestCase,
-) (tracinghelper.TestSpanTree, string) {
+) (tracinghelper.TestSpanTree, lib.EventMatchFunc) {
```

Comment:
does this break eventing-contrib? have you checked?

Reply:
it might also break knative-gcp

Reply:
Pretty sure it shouldn't (since it's not exported out of the package), but i'll throw a hold on and explicitly double check.
/hold

Reply:
eventing-contrib doesn't end up pulling this package into it's vendor at all, so safe.  gcp uses the package, but these functions aren't passed in from outside the package, so it's ok.